### PR TITLE
Optional ciphershare password prompt

### PR DESCRIPTION
**Pull request by nainiket-d**
Currently, it will only prompt for passwords while exporting documents from Ciphershare and not for other tasks like locking, deleting, etc. but this will be updated later once it is tested. 

This commit also has an updated OS info check for running Ciphershare as it runs differently on windows and Linux systems.
(See export document section on lines 89-95)

### CODE REVIEW

File: Automation/psi_ops_cms.py [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
 import jsonpickle
-
+import getpass
 
 #==============================================================================
 
 
 PSI_OPS_ROOT = os.path.abspath(os.path.join('..', 'Data', 'PsiOps'))
-PSI_OPS_DB_FILENAME = os.path.join(PSI_OPS_ROOT, 'psi_ops.dat')
+PSI_OPS_DB_FILENAME = os.path.join(PSI_OPS_ROOT, 'psi_ops_devops.dat')
```
Comment (adamkruger) [inline Automation/psi_ops_cms.py:32]: Please revert this change.

**Review by adamkruger — CHANGES_REQUESTED** [review]

File: Automation/psi_ops_cms.py (hunk 1 above)
Comment (nainiket-d) [reply Automation/psi_ops_cms.py:32]: Reverted 👍

File: Automation/psi_ops_cms.py [hunk 2]
```diff
@@ -87,21 +87,26 @@ def lock_document():
 
 
 def export_document(dest_filename):
-    cmd = 'CipherShareScriptingClient.exe \
-            ExportDocument \
+    if sys.platform in ['win32','cygwin']:
+        cmd = 'CipherShareScriptingClient.exe'
+    else:
+        cmd = 'wine CipherShareScriptingClient.exe'
+        os.remove(dest_filename)
```
[the remark this answers is not in the document]
Comment (nainiket-d) [reply Automation/psi_ops_cms.py:94]: os.remove removes the previous (created when running psi_ops.py) db file from the relative location (../Data/Psiops).
- The Ciphershare client overwrites the DB file “psi_ops.db” and works smoothly for **windows.**
- For **linux** and other *nix OS, Ciphershare exits with an error due to the existing file path.

Hence removing it with os.remove fixes the issue for linux and is not necessary for windows OS.
